Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/wooden_stake.yml

```diff
@@ -0,0 +1,49 @@
+- type: entity
+  name: wooden stake
+  parent: Spear
+  id: WoodenStake
+  description: Essential appliance for pitching tents and killing vampires.
+  components:
+  - type: Tag
+    tags:
+    - Spear
+    - WeaponMeleeStake
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Melee/wooden_stake.rsi
+    size: 2
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    attackRate: 1.5
+    damage:
+      types:
+        Piercing: 8
```

Comment:
How is this compering to other stuff consider you can build it for cheap
This vs glass shiv for say?

Reply:
OP, needs nerfing.

Reply:
I think glass shiv is 5.5 of slashing damage. And brand new crowbar from vendomat is 10 brute dmg.
But I'd also argue that wooden plank is less readily available than glass.

Reply:
Eh, I'll bring it down to 6. This thing should not be equal to Crusher Dagger.